2-configs/toml/toml-task.py

```diff
@@ -0,0 +1,33 @@
+import tomllib
+import os
+
+FILE_NAME = 'test.toml'
+
+
+def file_exist():
+    path = os.path.join(os.getcwd(), FILE_NAME)
+    if os.path.exists(path):
+        return True
+
+
+def read_toml_file(file):
+    try:
+        with open(file, "rb") as f:
+            data = tomllib.load(f)
+        return data
+    except Exception as e:
+        print("Can't read a file \n{}".format(e))
+        return None
```

Comment:
If you return here None, then in line 26:
```python
text = read_toml_file(FILE_NAME)
for info, user in text.items():
``` 
will occure error, as None does not have attribute items(). I would suggest to return empty dict()
```suggestion
        return dict()
```